### DISCUSSION TITLE
Disabling 'Yesterday/Alert lines in trends' doesn't hide yesterday in cards?

### DISCUSSION THREAD

**Discussion by chocmake** [breezy-weather/breezy-weather#306] [Q&A] [answered]
Maybe this setting does something else that I just haven't noticed but I expected the setting to disable the 'Yesterday' item in the weekday timelines but it's still present.

I've tried restarting the app but same result. I also have a weekday overview widget enabled, fwiw.

Using app version 4.5.1-beta on Android 13.

Posting to Discussions since I'm not sure what the setting is supposed to do.

Edit: since I put Precipitation first I hadn't noticed it toggles a faint grey line on the Temperatures tab. Though worth pointing out the lines are still displayed on the widget regardless of the setting state.

**Comment by papjul** [comment, marked as answer]
You are in the "Main screen settings", not in "Widgets settings", so it applies to main screen.

But I wouldn't mind applying it to widgets as well, if you would like to open a new Idea, on Issues ;)
